src/sentry/templates/sentry/error-page-embed.js

```diff
@@ -146,11 +150,16 @@
       document.removeEventListener('keydown', handleFocus);
     }
     this.element.parentNode.removeChild(this.element);
+    if (window.parent) {
+      window.parent.postMessage('reportdialog_closed', '*');
```

Comment:
```suggestion
      window.parent.postMessage('sentry_reportdialog_closed', '*');
```

Let's namespace to sentry to reduce possible collisions/confusions as much as possible

Reply:
How about even more `__sentry_reportdialog_closed__`?

Reply:
Even better!